src/disco/components/Addon.js

```diff
@@ -113,11 +114,13 @@ export class Addon extends React.Component {
       return (<a href="#" className="theme-image"
                  data-browsertheme={this.getBrowserThemeData()}
                  onBlur={this.resetPreviewTheme}
-                 onClick={this.handleClick}
+                 onClick={this.clickInstallTheme}
                  onFocus={this.previewTheme}
                  onMouseOut={this.resetPreviewTheme}
                  onMouseOver={this.previewTheme}>
-        <img src={previewURL} alt={sprintf(i18n.gettext('Preview %(name)s'), {name})} /></a>);
+        <img src={previewURL}
+          alt={sprintf(i18n.gettext('Hover to preview or click to install %(name)s'), {name})}
```

Comment:
I don't think we have indentation rules in eslint but I feel like the `alt` should line up with `src` or `src` should drop down to match `alt`.